test.sh

```diff
@@ -35,6 +35,25 @@ test_output() {
   ok
 }
 
+test_pipe() {
+  COMMAND="echo $1 | dick"
+  DICK="`echo $1 | dick`"
+  EXPECTED="$2"
+
+  echo -ne " - '\e[1m$COMMAND\e[0m' should output '\e[1m$EXPECTED\e[0m'"
+
+  if [[ "$?" != 0 ]]; then
+    fail "'$COMMAND' exit status was $?."
+    exit 1
+  fi
+
+  if [[ $DICK != $EXPECTED ]]; then
+    fail "Expected '$2', but got '$DICK'."
+    exit 1
+  fi
+  ok
+}
+
```

Comment:
I'd really like to "merge" this `test_pipe()` with `test_output()`. Ie: add a `stdio` argument to `test_output()`. This way, we can document the behavior when both sttdin and a `--lenght` parameter is passed (which one takes precedence)?